server/models/admin.sql

```diff
@@ -226,4 +226,18 @@ CREATE PROCEDURE zComputeAllInventoryStockQuantities (
   CLOSE inventory_cursor;
 END$$
 
+
+DROP PROCEDURE IF EXISTS  zUnpostRecord $$
+CREATE PROCEDURE zUnpostRecord(
+  IN _record_uuid BINARY(16)
+)
+BEGIN
+
+  INSERT INTO posting_journal 
+  SELECT * FROM general_ledger WHERE record_uuid = _record_uuid;
+
+  DELETE FROM general_ledger WHERE record_uuid = _record_uuid;
+
```

Comment:
I discussed this with Mr. Wayne and he gave me a good suggestion- we should probably write this into a log somewhere.

We actually have a table for this, but it is only for editing.  It's called `transaction_history`.  [Here it is](https://github.com/IMA-WorldHealth/bhima/blob/master/server/models/schema.sql#L2063).  Let's enhance it by adding a column to tell us if we are editing or unposting.